getting-started/vs-code-integration/dpl-wizard.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Configure Document Processing Libraries 
+page_title: Document Processing Libraries Integration
+description: Learn how to add a reference to the Telerik Document Processing Libraries(DPL) packages in an existing Telerik Ui for Blazor application using the Document Processing Libraries Wizard.
+slug: getting-started-vs-code-integration-dpl
+position: 5
+---
+
+# Configuring Document Processing Libraries
+
+This article demonstrates how to add a reference to the [Telerik Document Processing Libraries (DPL)](https://www.telerik.com/document-processing-libraries) packages in an existing Telerik UI for Blazor application using the Document Processing Libraries Wizard. The wizard is included in the [Telerik UI for Blazor Visual Studio Code extension](https://marketplace.visualstudio.com/items?itemName=TelerikInc.blazortemplatewizard) and enables you to easily add the required packages when working with the Telerik Document Processing Library.
+
+## Getting the Wizard
+
+To use the Document Processing Libraries Wizard, install the [Telerik UI for Blazor Visual Studio Code extension](https://marketplace.visualstudio.com/items?itemName=TelerikInc.blazortemplatewizard).
+
+## Starting the Wizard
+
+`Using the File Explorer Context Menu`
+
+To use **Document Processing Libraries Configuration Wizard** from the **VS Code**'s file explorer's context menu:
+ - Right-click on a file in the files tree.
+ - Select the `Telerik UI for Blazor: Add Document Processing Libraries`.
+
+  ![Configure Document Processing Wizard, Context Menu](images/DPLBlazor_ContextMenu.png)
+
+`Using the Code Editor Command Palette`
+
+To use **Document Processing Libraries Configuration Wizard** from the **VS Code** command palette:
+
+ - Open a `.razor` file or a `Razor` language in the code editor.
+ - Select the `Telerik UI for Blazor: Add Document Processing Libraries`.
+
+ ![Configure Document Processing Wizard, Command Palette](images/DPLBlazor_Pallete.png)
+
+After executing one of the options the wizard must appear like this: 
```

Comment:
`Execute` might not be the best word choice (due to the negative connotations around it on the one hand, and due to its specific nuance of "executing commands/code", on the other hand). Consider going with something like `After completing`, for example.

I would also place the content from here on in a new `##` section, named for example `## Running the Wizard` or `## Using the Wizard` to separate it from the instructions around "starting".